navigation/StackNavigators.js

```diff
@@ -16,6 +23,15 @@ const config = Platform.select({
   },
 });
 
+export const AuthStack = createStackNavigator(
+  {
+    Welcome: WelcomeScreen,
+    Login: LoginScreen,
```

Comment:
Since we're making both Log In and Sign Up 2 words across the board, would it make sense to have it be `LogIn` everywhere to match `SignUp`

Reply:
yup sgtm